src/frontend/web_application/src/components/Modal/style.scss

```diff
@@ -3,6 +3,14 @@
 
 $m-modal__overlay-background: rgba(0, 0, 0, .75);
 
+body {
+  &.ReactModal__Body--open {
+    // this is the class applied by ReactModal to body when modal is open
+    // add overflow: hidden to avoid scrolling on body then
```

Comment:
what if the modal is taller as screen height ?

Reply:
imo Modal should not be tallest than screen height and manage their own overflow, no?